tutorial.html

```diff
@@ -68,7 +68,7 @@ <h1>Tutorial</h1>
       <button id="info-button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Info</span><span class="caret"></span></button>
       <ul class="dropdown-menu" role="menu" aria-labelledby="dLabel">
         <li><a tabindex="-1" href="#tutorial-anchor" id="share-button">Share <u>l</u>ink</a></li>
-        <li><a tabindex="-1" href="#tutorial-anchor" id="desdm-button"><u>D</u>ownload image from DESDM</a></li>
+        <li><a tabindex="-1" href="#tutorial-anchor" id="desdm-button"><u>D</u>ownload image from NERSC</a></li>
```

Comment:
I think that we may want to stay away from NERSC, since there was some talk about standing up a version of the exposure checker at IN2P3.